src/main/java/core/basesyntax/BooleanExpression.java

```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return (((a ^ b) && (c ^ d)) || ((a ^ c) && (b ^ d)));
```

Comment:
```suggestion
        return (a ^ b) && (c ^ d) || (a ^ c) && (b ^ d);
```
don't overweight your code with redundant braces